package.json

```diff
@@ -55,6 +57,9 @@
     "bracketSpacing": true
   },
   "eslintConfig": {
-    "extends": "eslint-config-shared"
+    "extends": "eslint-config-shared",
+    "globals": {
+      "__ENV": "readonly"
```

Comment:
Could you move the "globals" config to to the eslint-config-shared package? That way all the eslint config is at one place and not split at different places

Reply:
ah, this was too specific to the k6 requirement so wasn't sure if it should be in the shared package. Will do!